puppet/modules/scaleway/templates/manage_snapshots.erb

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/ruby
+#
+# Copyright Foreman Maintainers 2018
+# GPL-3.0
+#
+# This ruby script use the Scaleway API key to
+# ensure that recent snapshots of a volume are taken,
+# and that only the most recent snapshots are kept
+
+# args
+#
+# name of the host to snapshot
+# api_key for auth
+# org_id is required by the api too
+
+SERVER  = <%= @name    -%>
```

Comment:
Should we make this a static file and read JSON/YAML files?

Reply:
We could. Server could be $1 in the cronjob I guess too

Reply:
That makes sense. The API key and org ID are probably good to not pass on the command line but server does make sense.